UITests/ClearPrivateDataTests.swift

```diff
@@ -13,7 +13,12 @@ class ClearPrivateDataTests: KIFTestCase, UITextFieldDelegate {
         webRoot = SimplePageServer.start()
     }
 
-    override func tearDown() {
+    override func afterEach() {
```

Comment:
Yeah. I used it because I wanted to make clear that this is part of the test that I wanted to execute after every test, rather than about getting each test back to a good state for the next test.